### PR TITLE
Change default version to 0.0.0

### LINKED ISSUE #357: Set default Ballerina version to 0.0.0
**Description:**
Currently, only with the ballerina-update-tool itself, we cannot run `bal dist` command since the user agent needs a version.
We have set default Ballerina version to empty string. We need to set it to `0.0.0`.

**Describe your task(s)**

**Related Issues (optional):**

**Suggested Labels (optional):**

**Suggested Assignees (optional):**

### PR DESCRIPTION
**Pull request by NipunaMadhushan**
## Purpose
For user agent, we need a Ballerina version and this will change the default Ballerina version from empty string to `0.0.0`.
This will fix the issue when we use only the ballerina-update-tool itself to pull a distribution.

Fixes #357

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

**Review by sm1990 — DISMISSED** [review]

File: src/main/java/org/ballerinalang/command/util/ToolUtil.java [hunk 1]
```diff
@@ -93,6 +93,12 @@ public static String getCurrentBallerinaVersion() {
                     setInstallerVersion(installerVersionFilePath);
                 }
             }
+            String ballerinaVersionFilePath = OSUtils.getBallerinaVersionFilePath();
+            if (!new File(ballerinaVersionFilePath).exists()) {
+                String defaultBallerinaVersion = "0.0.0";
```
Comment (keizer619) [inline src/main/java/org/ballerinalang/command/util/ToolUtil.java:98]: Lets move this to constant and use everywhere

File: src/main/java/org/ballerinalang/command/util/ToolUtil.java [hunk 2]
```diff
@@ -113,7 +119,7 @@ public static String getCurrentInstalledBallerinaVersion() {
             return getVersion(OSUtils.getInstalledConfigPath());
         } catch (IOException e) {
             //If we files does not exist it will be empty and update tool continues without a distribution
-            return "";
+            return "0.0.0";
```
Comment (keizer619) [inline src/main/java/org/ballerinalang/command/util/ToolUtil.java:122]: Use constant here as well

Comment (NipunaMadhushan) [reply src/main/java/org/ballerinalang/command/util/ToolUtil.java]: Addressed suggestion.

File: src/main/java/org/ballerinalang/command/util/ToolUtil.java [hunk 3]
```diff
@@ -93,6 +94,12 @@ public static String getCurrentBallerinaVersion() {
                     setInstallerVersion(installerVersionFilePath);
                 }
             }
+            String ballerinaVersionFilePath = OSUtils.getBallerinaVersionFilePath();
```
Comment (keizer619) [inline src/main/java/org/ballerinalang/command/util/ToolUtil.java:97]: We are not using this variable right?  Shall we avoid ?

Comment (NipunaMadhushan) [reply src/main/java/org/ballerinalang/command/util/ToolUtil.java:97]: Updated the PR. OSUtils.getBallerinaVersionFilePath() has been used in a below step as well.

**Review by keizer619 — APPROVED** [review]

**Pull request merged**